python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -424,13 +445,15 @@ def from_modules(
                 definitions to include on the returned asset group.
             executor_def (Optional[ExecutorDefinition]): An executor to include on the returned
                 asset group.
+            extra_source_assets (Optional[Sequence[SourceAsset]]): Source assets to include in the
+                group in addition to the source assets found in the modules.
 
         Returns:
             AssetGroup: An asset group with all the assets defined in the given modules.
         """
         asset_ids: Set[int] = set()
         asset_keys: Dict[AssetKey, ModuleType] = dict()
-        source_assets: List[SourceAsset] = []
+        source_assets: List[SourceAsset] = list(extra_source_assets or [])
```

Comment:
nit: could this just be a check.opt_list_param(...)?

Reply:
trying this out, it appears that `opt_list_param` / `opt_sequence_param` do not accept a default argument

Reply:
maybe missing something but why would you need a default argument? it would default to an empty list automatically if extra_source_assets was None, no?